api_service.py

```
@@ -2055,6 +2052,25 @@ async def startup_event():
+    # Start background processor AFTER server is ready to accept requests
+    # This prevents healthcheck failures during initialization
+    # Schedule background processor to start after a short delay to ensure server is fully ready
+    async def delayed_background_start():
+        """Start background processor after a delay to ensure server is ready"""
+        await asyncio.sleep(5)  # Wait 5 seconds for server to be fully ready
+        try:
+            logger.info("Starting background processor (delayed start after server ready)...")
+            start_background_processor()
+            logger.info("Background processor started successfully")
+        except Exception as e:
+            logger.error(f"Failed to start background processor: {e}")
```

Ruff (0.14.3)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)